packages/wrangler/templates/patch-console.js

```diff
@@ -0,0 +1,9 @@
+globalThis.console = new Proxy(globalThis.console, {
+	get(target, p, receiver) {
+		if (p === "log" || p === "debug" || p === "info") {
+			return (...args) =>
+				Reflect.get(target, p, receiver)(WRANGLER_WORKER_NAME, ...args);
+		}
+		return Reflect.get(target, p, receiver);
+	},
+});
```

Comment:
Perhaps we could wrap the logger instead of patching the console method directly.

(sorry this wasn't in the first review)

Reply:
I don't think I quite know what you mean—which logger?

Reply:
That's the Wrangler logger, whereas this code is patching the console global _inside_ a Worker, which is slightly different